### DISCUSSION TITLE
Will `set_formatter()` be deprecated?

### DISCUSSION THREAD

**Discussion by sfcheung** [davidgohel/flextable#584] [Q&A] [answered]
Hi all,

I need to do something in a package that cannot be easily done with the `colformat_*()` functions. Using `set_formatter()` can solve my problem. Then I found that, according to the help page of `set_formatter()`, "This function will be deprecated in favor of the ⁠colformat_*⁠ functions ...".

However, I noticed that some recent posts at GitHub (e.g., #572) still suggests using `set_formatter()`.

Will `set_formatter()` be deprecated as planned? Or will it stay in the package? If I use `set_formatter()`, I will use it in a package. Therefore, if it will be deprecated as planned, I need to find another solution.

Personally, I think `set_formatter()` is a good option for cases not covered by  `colformat_*()`. I used these formatters a lot but having a Plan B (`set_formatter()`) is a good thing.

-- Shu Fai

**Comment by davidgohel** [comment, marked as answer]
yeah, sorry, bad documentation, we won't deprecate `set_formatter()`. I like it a lot also and use it quite often! I'll fix the documentation

**Reply by sfcheung** [reply]
Thanks a lot for the good news! :)
